services/src/main/java/org/keycloak/authentication/requiredactions/RecoveryAuthnCodesAction.java

```diff
@@ -160,11 +160,15 @@ public boolean isSupported(Config.Scope config) {
 
     @Override
     public List<ProviderConfigProperty> getConfigMetadata() {
-        return List.copyOf(CONFIG_PROPERTIES);
+        return Stream.concat(
+                List.copyOf(CONFIG_PROPERTIES).stream(),
+                List.copyOf(RequiredActionFactory.MAX_AUTH_AGE_CONFIG_PROPERTIES).stream()
```

Comment:
```suggestion
                super.getConfigMetadata()
```

Was just thinking that it can be better to call `super` here to make sure that if something changes in the `RequiredActionProvider`, this one is automatically updated as well.